Dockerfile

```diff
@@ -12,24 +12,17 @@ WORKDIR /app/
 COPY requirements.txt requirements.txt
 RUN pip install --upgrade pip && pip install setuptools -U && pip install --user -r requirements.txt
 
-# Install a custom OPAL, if requested
-COPY custom /custom
-
-RUN if [ -f /custom/custom_opal.tar.gz ]; \
```

Comment:
This shouldn't be removed, or be replaces with `opal-client` as a requirement.
If you use a requirement, keep in mind we have to release a new OPAL to pypi before we can release the sidecar.

Reply:
Last time we've talked we decided to add opal-client to requirements, remember?
I do not have any problem also to not remove it, what ever you choose, just let me know.

Reply:
I remember raising the same concern last time as well.
It's better to have it as a requirement, I'm just not in the mood to release a new OPAL version 😛 - maybe we should discuss this with @obsd

Reply:
I can make it with opal @ currently until the new version of opal public would be release